### DISCUSSION TITLE
AI Model is not available in your region

### DISCUSSION THREAD

**Discussion by thomaszdxsn** [zed-industries/zed#16556] [Feature Requests]
![CleanShot 2024-08-21 at 07 11 54](https://github.com/user-attachments/assets/4088f69e-4e0d-4697-aa54-b9999562d1dd)

Can I resolve it by use a proxy?

**Comment by SomeoneToIgnore** [comment]
Certain models put restrictions on list of countries they can be used in, some of that is dictated by the US directives — and since Zed partners with some of the model providers, it seems impossible to advice on how to overcome the "issue" or host such discussions.
